acceleration: 2
max_joint_vel: 0.2
max_cartesian_vel: 0.2
sampling_duration: 0.1

simulation: false

n_joints: 9

max_joint_limit: [100,   100,  100,  100, 100,  0.4, 1.37, 1.4529, 3.14]
min_joint_limit: [-100, -100, -100, -100, -100, -1.3, 0.1,  0.1,   -3.14]

PID_p_values: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
PID_i_values: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
PID_d_values: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

arm_joint_state_topic: /arm5e/joint_state_angle_fixed
arm_joint_command_topic: /arm5e/command_angle

vehicle_tf: girona500
world_tf: world
vehicle_command_topic: /cola2_control/body_velocity_req



chain_names: [auv_arm, auv]



auv_arm/joint_1: [TransX]
auv_arm/joint_2: [TransY]
auv_arm/joint_3: [TransZ]
auv_arm/joint_4: [RotZ, 0.0275, 0.0, 0.66, -1.5708]
auv_arm/joint_5: [RotZ, -0.18052, 0.0, 0.0, -1.5708]
auv_arm/joint_6: [RotZ, 0.08052, 1.5708, 0.0, 0.0]
auv_arm/joint_7: [RotZ, 0.44278, 0.0, 0.0, 0.139277274]
auv_arm/joint_8: [RotZ, -0.083, 1.5708, 0.0, 1.972222055]
auv_arm/joint_9: [RotZ, 0.0, 0.0, 0.56138, 0.0]


auv_arm/chain_joint_relation: [0, 1, 2, 3, 4, 5, 6, 7, 8]
auv_arm/max_cartesian_limit: [20,20,20, 20, 20, 20]
auv_arm/min_cartesian_limit: [-20,-20,-20, -20, -20, -20]


auv/joint_1: [TransX]
auv/joint_2: [TransY]
auv/joint_3: [TransZ]
auv/joint_4: [RotZ]

auv/chain_joint_relation: [0, 1, 2, 3]
auv/max_cartesian_limit: [20,20, 3.5, 6.28, 6.28, 6.28]
auv/min_cartesian_limit: [-20,-20,-20, -6.28, -6.28, -6.28]




multitask_priority: [joints_ros_multi]



joints_ros_multi/tasks: [joints_ros]
joints_ros_multi/joint_priority_1: [0,1,2,3,5,6,7,8]



joints_ros/chain: auv_arm
joints_ros/cartesian: False
joints_ros/mask_cartesian: [1, 1, 1, 1, 1, 1]
joints_ros/mask_joint: [1,1, 1, 1,0,1,1,1,1]
joints_ros/goal_type: JointsROS
joints_ros//goal/joints_topic: /requested_joint_config
joints_ros/frame_inertial: True




